### ISSUE TITLE
[Warlock] Pet Spawning with Low Health

### ISSUE THREAD

**Issue by Skipslegdays** [dalaranwow/dalaran-wow#5891]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Pets can sometimes spawn with less than 10% health and mana after dismounting. It does not happen every time and I have not figured out what causes it. It was an issue when I played here four years ago as well.

**Expected behaviour**: Tell us what should happen instead
They should spawn with the same health and mana as when you mounted

**Steps to reproduce the problem**:

1.  Have a pet summoned with full health and mana in Battleground
2.  Mount
3.  Dismount

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Skipslegdays** [comment]
Still happening as of April 2021. I am using the celestial horse mount, that could be a cause.

**Comment by Skipslegdays** [comment]
Pretty sure it is because of the Celestial Steed (or maybe any other ground+flying mount). Swapped mounts and it stopped happening.
